src/modules/map/components/openlayers/OpenLayersMap.vue

```diff
@@ -33,11 +33,25 @@
         />
         <!-- Adding highlighted features -->
         <OpenLayersHighlightedFeature
-            v-for="(feature, index) in highlightedFeatures"
+            v-for="(feature, index) in selectedFeatures"
             :key="`${index}-${feature.id}`"
```

Comment:
We don't need to change this. But I'm curious. Is the `feature.id` necessary?

Reply:
not required per say, but `v-for` wants to have unique keys for each element of the collection. So the easier way is to ask each feature to have an ID (or to build it through the index of the loop and some other string?!)

Reply:
I don't know enough yet about how Vue handles its components. What I wonder is if there is a situation where the index of an object changes. I guess that would be possible with splice or something similar.

Reply:
yep that could, the goal here was to have a unique identifier for each features, and as stated in Feature class JSDoc, it's possible to have to distinct features (unrelated, from different layers or sources) that have the same ID, and that would be valid.
So to make sure it doesn't happen, I've appened the index (of the loop) to the feature ID, I could also use the layer id (as now I've access to it in the feature metadata)

Reply:
:+1: